src/plugins/intel_cpu/src/nodes/fullyconnected.cpp

```diff
@@ -1004,7 +1017,8 @@ bool FullyConnected::canBeExecutedInConv1x1() const {
         widthInConv = srcDims[inRank - 2];
         K = srcDims[inRank - 1];
         N = weightDims[0];
-
+        if (weightMemPtr->getSize() >= (16 * 1 << 20))
```

Comment:
Lets add comment that we are disabling Conv1x1 to avoid different weights layout usage and as a consuquence we decrease memory consumptuion. It also it worth mention that condition is limited 16 MB (bit shifting is not so easy to understand).

Reply:
Applied.